core/requirements/kvp-execute/REQ_bbox-crs-input-value.adoc

```diff
@@ -12,13 +12,13 @@ identifier:: /req/kvp-execute/bbox-crs-input-value
 
 [.component,class=part]
 --
-A bbox-crs input parameter SHALL have the following characteristics:
+A `inputCrs` input parameter SHALL have the following characteristics:
 
 [source,yaml]
 ----
 include::../../../openapi/parameters/common-crs/bbox-crs.yaml[]
 ----
 
-where the token `{bbox-input-name}` represents the identifier of a bbox input parameter and the token  `{bbox-input-name}-crs` represents the identifier of a corresponding input that indicates which CRS is being used for the coordinates of the `{bbox-input-name}` parameter.
+where the token `{bbox-input-name}` represents the identifier of a bbox input parameter and the token  `inputCrs[{bbox-input-name}]` represents the identifier of a corresponding input that indicates which CRS is being used for the coordinates of the `{bbox-input-name}` parameter.
```

Comment:
Isn't it supposed to be the other way around? 
A bbox input named `test-bbox` would provide its CRS as `test-bbox[crs]=...`.
This would map well with `test-bbox[value]=...` for the qualified input value.
However, we imply that `test-bbox == test-bbox[value]` for ease of use.

In this case, the `inputCrs` name and the "variable" `{bbox-input-name}` are misleading. 

The text earlier should be "A `{bbox-input-name}` input parameter ..."

Reply:
Done.